Project-2/Tr2.0

```diff
@@ -0,0 +1,237 @@
+Project 2
+
+Your second project will require you to answer each of the 10 questions below.  You will be expected to open a pull request with your initial answers by the second class meeting, giving you one week to work on these problems. You and your peers will then have one week to work together to refine your respective initial answers, so they are ready for final submission. Once your pull requests have been reviewed and merged to the development branch, I will review them, then merge to the master branch. 
+
+```
+Tip #1: Carefully study the Baader, et. al. selections assigned on bisimulation; it is deceptively subtle, and quite powerful. 
+Tip #2: Google is still your friend. So is stackexchange...
+Tip #3: Work together to solve these problems, even for initial submissions and when you do, document this in github. 
+Tip #4: Work together as a team. 
+```
+
+Symbols: ∀ ∃ ⊔ ⊓ ⊧ ⊭ ⊦ ⊬ ⊏ ⊐ ⊑ ⊒ ≡ ¬ → ∨ ∧ ¯
+
+
+1. Let V be a vocabulary of ALCI consisting of a role name "P". Interpret part_of as "x is a part of y". Using this role name, define the following formulas in this language:
+
+  (a)  PP that says that x is a proper part of y
+
+  PP ≡ P⊓~P¯
+
+  (b)  iPP that says that y is a proper part of x
+
+  iPP ≡ PP¯
+
+  (c)  iP that says that y has x as part 
+
+  iP ≡ P¯
+
+  (d)  O that says that x overlaps y
+
+  O ≡ ∃P¯.(∃P)
+
+  (e)  D that says that x and y are disjoint 
+
+  D ≡ ~O
+
+
+
+2. Use your axioms from question 1 as the basis of an ALCI T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+
+
+  Tbx = {PP = ∃P.~P         (Tbx. 1)
+
+        iPP = PP¯            (Tbx. 2)
+
+        iP = P¯              (Tbx. 3)
+
+        O = ∃P¯.(∃P)        (Tbx. 4)
+
+        D = ~O               (Tbx. 5)
+
+
+  Abx = {(Philosophy-department, UB): part_of           (Abx. 1)
+
+        (UB, City-of-Buffalo): part_of                  (Abx. 2)
+
+        (City-of-Buffalo, UB): inverse_part_of          (Abx. 3)
+
+        (UB, Philosophy-department): inverse_part_of    (Abx. 4)
+
+        (UB, City-of-Buffalo): overlap                  (Abx. 5)
+
+        (UB, Wegman's): disjoint                        (Abx. 6)
+
+
+ΔI = {w, x, y, z},
+
+  UB = w
+
+  City-of-Buffalo = x
+
+  Philosophy-department = y
+
+  Wegman's = z
+
+
+  Part_of = {(w,x),(y,w),(z,x)}
+  
+  Inverse_Part_of = {(x,w),(y,w),(x,z)}
+  
+  Overlap = {(w,x),(y,w),(z,x)}
+  
+  Disjoint = {(w,z),(y,z)}
+
+
+
+3. Translate the following first-order logic axioms into ALCI: 
+
+(a) ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+∃R.∀R ⊓ ∀R
+
+
+(b) ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+∃R¯.∃R ⊓ ∃R
+
+
+(c) ∀y(R(x, y) → ∃x(R(y, x) ∧ ∀y(R(x, y) → A(y))))
+
+∀R.∃R.∀R.A
```

Comment:
Your formula says all x related to only something that is related to something that is only related to some A. Is this what you intended?